migrations/Version202109281632856186_taoLti.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoLti\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+//use OAT\Library\Lti1p3Ags\Factory\Score\ScoreFactory;
+//use OAT\Library\Lti1p3Ags\Service\Score\Client\ScoreServiceClient;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\tao\scripts\update\OntologyUpdater;
+use oat\taoLti\models\classes\LtiAgsScoreService;
+
+final class Version202109281632856186_taoLti extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        OntologyUpdater::syncModels();
+
+        $this->getServiceManager()->register(
+            LtiAgsScoreService::SERVICE_ID,
+            new LtiAgsScoreService(
+                // option are commented, coz there is an error on migration "Serialization of 'Closure' is not allowed"
```

Comment:
That's because the current service manager tries to php-serialize the injected objects.
I don't think you'll be able to register a service like that.

You may try extending your `LtiAgsScoreService` from the `InjectionAwareService` instead, but I'm not quite sure if that's gonna help either.

If all fails, please consider executing a simple file copy from `config/default/LtiAgsScoreService.conf.php` to the TAO instance `config/<extension>/LtiAgsScoreService.conf.php` as part of this migration.

Reply:
done